examples/cc-template/WORKSPACE

```diff
@@ -0,0 +1,36 @@
+# load the http_archive rule itself
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+# load rules_nixpkgs
+http_archive(
+    name = "io_tweag_rules_nixpkgs",
+    strip_prefix = "rules_nixpkgs-9f08fb2322050991dead17c8d10d453650cf92b7",
+    urls = ["https://github.com/tweag/rules_nixpkgs/archive/9f08fb2322050991dead17c8d10d453650cf92b7.tar.gz"],
+    sha256 = "46aa0ca80b77848492aa1564e9201de9ed79588ca1284f8a4f76deb7a0eeccb9",
+)
```

Comment:
This does point to the commit at the tip of 0.9.0; for whatever reason, when I switch this to
```
http_archive(
    name = "io_tweag_rules_nixpkgs",
    strip_prefix = "rules_nixpkgs-0.9.0",
    urls = ["https://github.com/tweag/rules_nixpkgs/archive/refs/tags/v0.9.0.zip"],
    sha256 = "a8f4cc94757f19a0a1ce95245f7a87f294cfbce9859ed55ffaeeacf6f0d0af16",
)
```
I get errors about illegal symlinks in the `.bazelrc`:
```
ERROR: no such package '@rules_nixpkgs_python//': java.io.IOException: Error extracting /home/w/.cache/bazel/_bazel_w/ef63e3bfe165b0613795ad5212904c1a/external/rules_nixpkgs_python/temp15388650320627280841/v0.9.0.zip to /home/w/.cache/bazel/_bazel_w/ef63e3bfe165b0613795ad5212904c1a/external/rules_nixpkgs_python/temp15388650320627280841: Zip entries cannot refer to files outside of their directory: v0.9.0.zip has a symlink .bazelrc pointing to ../../core/.bazelrc
```

Reply:
Hmm, interesting. Does it work if you use the `.tar.gz` instead? Maybe that's just a Zip limitation?
Nonetheless, it's bad to have broken release artifacts. @benradf I think this can be fixed by cherry-picking the later changes that replace the `.bazelrc` symlinks by files with `%workspace%` relative imports. Would you have bandwidth to create a patch release to fix this?
You can use `urls = ["file:///..."]` syntax to point `http_archive` to local files, which should make local testing easier.

Reply:
@aherrmann @benradf I don't see a patch release in the releases list; has one been made yet? The workaround with the .tar.gz archive does indeed work, so I'm going with that for now.

Reply:
No, unfortunately I've not had time to make a patch release yet.

Reply:
Got it. I'll resolve this thread; feel free to make an issue to track this (since the zip for 0.9.0 is still broken).

Reply:
I created #373 to track this.